.eslintrc.js

```diff
@@ -34,7 +34,8 @@ module.exports = {
     'react/jsx-props-no-spreading': 'off',
     'simple-import-sort/imports': 'error',
     'simple-import-sort/exports': 'error',
-    '@typescript-eslint/prefer-nullish-coalescing': 'warn',
+    '@typescript-eslint/prefer-nullish-coalescing': 'error',
+    'no-magic-numbers': 'error',
```

Comment:
Oh cool, I didn't know about this rule. 👍  I added to https://near-foundation.atlassian.net/jira/software/c/projects/ENG/boards/1?modal=detail&selectedIssue=ENG-156&quickFilter=1